apps/editor.planx.uk/src/ui/editor/SelectInput/SelectInput.tsx

```diff
@@ -41,11 +41,14 @@ const Root = styled(Select)(({ theme }) => ({
     alignItems: "center",
     justifyContent: "flex-start",
   },
-  [`&.${classes.icon}`]: {
-    right: theme.spacing(1),
-    color: "rgba(0,0,0,0.25)",
+  // Match caret in MultipleSelect component
+  [`& .${classes.icon}`]: {
+    right: theme.spacing(0.85),
+    bottom: theme.spacing(0.5),
+    color: theme.palette.primary.main,
+    fontSize: 35,
```

Comment:
Noticed a small visual issue with the icon here when working on integrating this in Questions / Checklists - 

**Before**
<img width="522" height="379" alt="Image" src="https://github.com/user-attachments/assets/dd8abee1-120f-477c-84bb-c43891dc17c3" />

**After**

<img width="489" height="369" alt="Image" src="https://github.com/user-attachments/assets/d0f734fd-4782-4722-8a3c-4f155121fd68" />